Documentation/guides/messages/xa1031.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Xamarin.Android error XA1031
+description: XA1031 error code
+ms.date: 10/10/2022
+---
+# Xamarin.Android error XA1031
+
+## Example messages
+
+```
+The 'AndroidHttpClientHandlerType' has an invalid value of 'xxx' please check your project settings.
+```
+
+## Solution
+
+Open your project settings in Visual Studio and check that the
+'AndroidHttpClientHandlerType' has a valid value. Alternatively
+you can edit your csproj directly and change the 'AndroidHttpClientHandlerType'.
```

Comment:
I don't think we have this setting in the VS project options pages for .NET 6 projects. The Mono team wanted to "hide" it so only more advanced users would be using it.

Maybe we just reword this to edit .xml the project file?